src/App/App.stories.tsx

```diff
@@ -11,4 +12,48 @@ export default {
 
 const Template: Story = args => <App {...args} />;
 
+// TODO: maybe replace built-in mocks with msw after adding it
```

Comment:
If it works, we don't need to have MSW.

Reply:
I personally did not like how cypress handles graphql mocks so I thought maybe we use MSW for storybook, dev env and cypress.

Reply:
it doesn't add much of a benefit but we would have consistency (using one tool for mocking)
the difference is that for cypress we have to do [extra work](https://docs.cypress.io/guides/end-to-end-testing/working-with-graphql#Alias-multiple-queries-or-mutations) for cypress to capture graphql requests   

but it's not a big deal so yeah, we can forget about MSW if you agree

Reply:
But is not a good idea to have duplication; if MSW can eliminate that duplication, let's use it.

Reply:
I'm not sure tho maybe we can ignore MSW for now I'll try it out in the MSW ticket, if it went good (it did add good values) I'll do a pr if not then I'll inform you too drop the ticket

Reply:
Agree